heareval/score.py

```diff
@@ -239,6 +239,28 @@ class EventBasedScore(SoundEventScore):
             "percentage_of_length": 0.5,
         },
     ),
+    "event_onset_50ms_fms": partial(
+        EventBasedScore,
+        name="event_onset_50ms_fms",
+        score="f_measure",
+        params={
+            "evaluate_onset": True,
+            "evaluate_offset": False,
+            "t_collar": 0.05,
+            "percentage_of_length": 0.5,
+        },
+    ),
+    "event_onset_offset_50ms_20perc_fms": partial(
+        EventBasedScore,
+        name="event_onset_offset_50ms_20perc_fms",
```

Comment:
In MAESTRO paper (and our doc), they use fms + prc + rcl. Is it possible (and not too hard) to compute all three without running sed_eval three times (which is slow)? Maybe we could even run EventBasedScore just once for both onset f+p+r and onset+offset f+p+r. Like one sed_eval for all six?

Reply:
#333